web/modules/custom/joinup_invite/src/Form/InviteForm.php

```diff
@@ -0,0 +1,266 @@
+        ],
+        '#empty' => 'No users found.',
+        '#weight' => 15,
+      ];
+
+      if (!empty($rows)) {
+        $form['results_container']['submit'] = [
+          '#type' => 'submit',
+          '#value' => t('Add facilitators'),
+          '#weight' => 20,
+        ];
+      }
+    }
+
+    return $form;
+  }
+
+  /**
+   * Returns the header of the main table.
+   *
+   * @return array
+   *   The header.
+   */
+  protected function getHeader() {
+    return [['data' => 'Personal Info']];
+  }
+
+  /**
+   * Return the rows.
+   *
+   * @param string $filter
+   *   The filter.
+   *
+   * @return array
+   *   The rows of the tableselect.
+   */
+  private function getRows($filter) {
+    if ($user = user_load_by_mail($filter)) {
+      $users = [$user];
+    }
+    else {
+      $results = $this->entityTypeManager->getStorage('user')->getQuery('OR')
+        ->condition('mail', '%' . $filter . '%', 'LIKE')
+        ->condition('field_user_first_name', '%' . $filter . '%', 'LIKE')
+        ->condition('field_user_family_name', '%' . $filter . '%', 'LIKE')
+        ->execute();
+      /** @var \Drupal\user\UserInterface[] $users */
+      $users = User::loadMultiple($results);
+    }
+
+    $rows = [];
+    foreach ($users as $user) {
+      $name = $this->getAccountName($user);
+      $rows[$user->id()] = [
+        'title' => ['data' => ['#title' => $name]],
+        0 => $name,
+      ];
+    }
+    return $rows;
+  }
+
+  /**
+   * Returns a full name of the user.
+   *
+   * @param \Drupal\user\UserInterface $user
+   *   The user object.
+   *
+   * @return \Drupal\Core\StringTranslation\TranslatableMarkup
+   *   A string version of user's full name.
+   */
+  protected function getAccountName(UserInterface $user) {
+    $first_name = empty($user->get('field_user_first_name')->first()->value) ? '' : $user->get('field_user_first_name')->first()->value;
+    $family_name = empty($user->get('field_user_family_name')->first()->value) ? '' : $user->get('field_user_family_name')->first()->value;
+
+    return $this->t('@name (@email)', [
+      '@name' => implode(' ', [$first_name, $family_name]),
+      '@email' => $user->getEmail(),
+    ]);
+  }
+
+  /**
+   * Form submission handler for the filter submit.
+   *
+   * @param array $form
+   *   An associative array containing the structure of the form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The current state of the form.
+   */
+  public function filterSubmit(array &$form, FormStateInterface $form_state) {
+    $form_state->setRebuild();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validateForm(array &$form, FormStateInterface $form_state) {
+    parent::validateForm($form, $form_state);
+    $triggering_element = $form_state->getTriggeringElement();
+    $users = array_filter($form_state->getValue('users'));
+    if ($triggering_element['#name'] === 'op' && $triggering_element['#value'] !== 'Filter' && empty($users)) {
```

Comment:
The best way is to actually set a `#name` on the button so you can match on it safely.